### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by BrettRToomey**

### CODE REVIEW

**Comment by BrettRToomey** [comment]
Retabbed all the files with Vim ❤️

File: README.md [hunk 1]
```diff
@@ -7,6 +7,38 @@ This collection of templates is related to models and automating their conversio
 ## Model
 Automatically generates an initializer and an enum for MySQL enum types.
 
+Example:
+```swift
+final class User: Model {
```
Comment (steffendsommer) [inline README.md:12]: Missing annotation with `model` this should be introduced somewhere as well. Sourcery will only pick up types with the `model` annotation.

File: README.md [hunk 2]
```diff
@@ -17,6 +49,45 @@ Automatically generates an initializer and an enum for MySQL enum types.
 ## Preparation
 Generates a list of database keys, automates `prepare` and `revert` functions.
 
+Example:
+```swift
+final class User: Model {
```
Comment (steffendsommer) [inline README.md:54]: Missing `model` annotation

File: README.md [hunk 3]
```diff
@@ -34,6 +105,41 @@ Generates a list of database keys, automates `prepare` and `revert` functions.
 ## RowConvertible
 Automates `init (row: Row)` and `makeRow` boilerplate.
 
+Example:
+```swift
+final class User: Model {
```
Comment (steffendsommer) [inline README.md:110]: Missing `model` annotation

File: README.md [hunk 4]
```diff
@@ -43,6 +149,40 @@ Automates `init (row: Row)` and `makeRow` boilerplate.
 ## NodeRepresentable
 Generates a list of node keys and `makeNode(in context: Context?)`.
 
+Example:
+```swift
+final class User: Model {
```
Comment (steffendsommer) [inline README.md:154]: Missing `model` annotation

File: README.md [hunk 5]
```diff
@@ -52,6 +192,46 @@ Generates a list of node keys and `makeNode(in context: Context?)`.
 ## JSONConvertible
 Generates a list of JSON keys, `init(json: JSON)` and `makeJSON`.
 
+Example:
+```swift
+final class User: Model {
```
Comment (steffendsommer) [inline README.md:197]: Missing `model` annotation

**Review by steffendsommer — CHANGES_REQUESTED** [review]

**Review by steffendsommer — APPROVED** [review]

**Pull request merged**